.github/workflows/release.yml

```diff
@@ -4,6 +4,9 @@ on:
   workflow_dispatch:
     branches:
       - master
+  push:
```

Comment:
Some thoughts about this push trigger:

There are automated pushes to master, like from dependabot or semantic-release itself. The latter adds a `[skip ci]` to its commit descriptions. dependabot commits _do_ trigger the CI and semantic-release will be run, but then the commit analyzer won't find commits that are worth a new release. Example:
![image](https://user-images.githubusercontent.com/12255735/126904344-0381d732-35f9-4ef1-859f-7e7d3bdc152f.png)

That would be wasted runtime, especially with the truckloads of commits from ~dospamalot~ dependabot 🤡. Adding `[skip ci]` to dependabot commit descriptions would save the load. But I doubt it is desirable to skip the CI build for dependabot.

Reply:
@code-shambles  Thanks for the suggestion. What about this [kind of dependabot commit](https://github.com/foobaragency/cf-migrations/pull/111), should they not cause a patch release? 

Is that the reason why you did it to be a manual workflow run?

Reply:
I don't think that `chore` commits trigger a new release. https://github.com/semantic-release/semantic-release#commit-message-format